src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -135,16 +133,15 @@ class LocalBackend extends Backend with LazyLogging {
    */
   private def stageWorkflowInputs(descriptor: WorkflowDescriptor): HostInputs = {
     val hostInputsPath = Paths.get(hostExecutionPath(descriptor).toFile.getAbsolutePath, "workflow-inputs")
-    descriptor.actualInputs map {stageInput(_, hostInputsPath)}
+    descriptor.actualInputs.map {case(name, value) => name -> stageWdlValue(value, hostInputsPath)}
```

Comment:
If you're going to insist on using this in between styles construct at least don't add dots.